### DISCUSSION TITLE
Where to find open sorece community? how to connect with them

### DISCUSSION THREAD

**Discussion by fardin-developer** [community/community#128198] [Other Feature Feedback, Questions, & Ideas]
How and where to find open sorece projects

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @fardin-developer  ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. 

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ebndev** [comment]
Hi @fardin-developer, thanks for being part of the GitHub Community!

For more information, you can visit our docs on [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github) and [How to choose (and contribute to) your first open source project](https://github.com/collections/choosing-projects) for a collection of a few projects to get you started.

Best of luck on your coding journey and can’t wait to see your contributions 🌟

**Comment by Mored241** [comment]
You can start with Stackoverflow, Dev.to, Github (here), X(twitter), Reddit and even some facebook's Groups...
